exercises/04_all_zero.rs

```diff
@@ -6,9 +6,7 @@ use creusot_contracts::*;
 pub fn all_zero(v: &mut Vec<u32>) {
     let mut i = 0;
     let old_v = snapshot! { v };
-    // Until https://gitlab.inria.fr/why3/why3/-/merge_requests/667 is merged
-    // the following invariant is needed to allow Why3 to remember prophecies dont change
-    #[invariant(proph_const, ^v == ^old_v.inner())]
```

Comment:
Keep in mind, this is actually now needed _again_ by the change to coma. At least until 0.2 comes out (soon ish).

Reply:
Interesting, so that’s why I needed `#[invariant(^*old_v == ^v)]`? The old syntax with `proph_const` didn’t seem to work …